backend/benefit/applications/tests/test_benefit_aggregation.py

```diff
@@ -33,7 +34,7 @@
             ],
             BenefitType.SALARY_BENEFIT,
             NO_APPRENTICESHIP,
-            400,
+            "Benefit can not be granted before 24-month waiting period expires",
```

Comment:
The source text you are comparing against is a translatable string and here it is not. If someone adds translations for the source string then wouldn't this possibly change to something else?

Reply:
I'll check, maybe need to force the locale explictly